### PR TITLE
[MSCTFIME] Add new Portuguese (pt-PT) translation.

### PR DESCRIPTION
**Pull request by zecarlos1957**
## Purpose

Add new Portuguese (pt-PT) translation.

### CODE REVIEW

**Review by HBelusca — APPROVED** [review]

File: dll/ime/msctfime/lang/pt-PT.rc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    IDS_RETRY            "&Retroceder"
+    IDS_IGNORE           "&Ignorar"
+    IDS_YES              "&Sim"
+    IDS_NO               "&Não"
+
+    IDS_ENTER            "Enter"
+    IDS_FINALIZE_STRING  "Finalizar cadeia"
+    IDS_CONVERSION       "Conversão"
+END
+
```
Comment (binarymaster) [inline dll/ime/msctfime/lang/pt-PT.rc:24]: Excess line at the end.

Comment (zecarlos1957) [reply dll/ime/msctfime/lang/pt-PT.rc:24]: At https://reactos.org/wiki/Translation_Introduction
it says: Leave an end of file line (a blank line) at the end of your rc file for ease of building.

Comment (binarymaster) [reply dll/ime/msctfime/lang/pt-PT.rc:24]: Yes, but you have two lines, one of which is excessive.

Comment (zecarlos1957) [reply dll/ime/msctfime/lang/pt-PT.rc]: You're right. My apologies.

**Review by binarymaster — APPROVED** [review]

**Review by JoachimHenze — APPROVED** [review]
Approved. No pt-BR yet, SUBLANG_NEUTRAL is ok.

**Pull request merged**